src/allotropy/parsers/appbio_quantstudio/appbio_quantstudio_data_creator.py

```diff
@@ -0,0 +1,77 @@
+from allotropy.parsers.appbio_quantstudio.appbio_quantstudio_calculated_documents import (
+    build_quantity,
+    iter_calculated_data_documents,
+)
+from allotropy.parsers.appbio_quantstudio.appbio_quantstudio_structure import (
+    AmplificationData,
+    Data,
+    Header,
+    MeltCurveRawData,
+    MulticomponentData,
+    RawData,
+    Result,
+    WellList,
+)
+from allotropy.parsers.lines_reader import LinesReader
+from allotropy.parsers.utils.values import try_str_from_series_or_none
+
+
+def create_data(reader: LinesReader) -> Data:
```

Comment:
Please go ahead and put this as Data.create method in the structure file.  

This function is not over-large, and I think after this refactor there will be some other patterns we can identify to simplify further :D 

For future reference, if we were going to break up a file, I would lean towards just breaking up large Structure classes into their own files. But in this case, I think it's OK to be in a single file.

Reply:
I tried to move this function into a Data.create function. But I was not able to fix a circular reference issue that results.

The problem is that this method creates the calculated data documents and those use the other structure classes
So, If I move this code to the structure a circular reference is created.

Reply:
Ah ok, I thought it was just due to size. I still think we can make it work, but OK with merging this and taking a crack at it later. 👍